source/includes/examples/tf-staging-prod-complete/main.tf

```diff
@@ -177,48 +201,54 @@ resource "mongodbatlas_privatelink_endpoint" "test" {
   }
 }
 
-# AWS ONLY- remove for other cloud providers: Enable BYOK encryption
-resource "mongodbatlas_cloud_provider_access_setup" "setup_only" {
+# AWS ONLY - Enable BYOK encryption
```

Comment:
nit: Since you have now defined example within a directory you can consider splitting `main.tf` into separate files for different functionality (e.g. BYOK encryption, private link). This might be useful for more specific referencing. From terraform point of view having configurations all in single file vs multiple files is the same.

Reply:
Good call! Split out a couple functionalities into different files